Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR /app
 
 COPY poetry.lock pyproject.toml /app/
 
-RUN pip install poetry && \
+RUN pip install -U pip poetry && \
```

Comment:
Since we use pip to install plugins at runtime, it's good for it to be up-to-date